extract_verbs.sh

```diff
@@ -3,20 +3,91 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+
+# Check if the argument is null
+if [ ! -n "$1" ] ;then
+    echo "No language code entered!"
+    echo "Usage:"
+    echo "./extract_verbs.sh languageCode(e.g it OR italian for Italian)"   
+    exit
+fi
+# Get the entered language code
+langCode=$1
```

Comment:
Just a minor comment: if you make this variable uppercase, it will improve the code readability